
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../CKM-001-%20Reviewing-an-Archetype-on-CKM/">
      
      <link rel="icon" href="../images/apperta-logo.png">
      <meta name="generator" content="mkdocs-1.4.2, mkdocs-material-9.1.5">
    
    
      
        <title>Pharmacogenetics Data Modelling - A Step Towards Personalised Prescribing - Apperta CKM - NHS Pharmacogenetics (PGx) project</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.7a7fce14.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.a0c5b2b5.min.css">
      
      

    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue-grey" data-md-color-accent="teal">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#pharmacogenetics-data-modelling-a-step-towards-personalised-prescribing" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Apperta CKM - NHS Pharmacogenetics (PGx) project" class="md-header__button md-logo" aria-label="Apperta CKM - NHS Pharmacogenetics (PGx) project" data-md-component="logo">
      
  <img src="../images/apperta-logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Apperta CKM - NHS Pharmacogenetics (PGx) project
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Pharmacogenetics Data Modelling - A Step Towards Personalised Prescribing
            
          </span>
        </div>
      </div>
    </div>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Apperta CKM - NHS Pharmacogenetics (PGx) project" class="md-nav__button md-logo" aria-label="Apperta CKM - NHS Pharmacogenetics (PGx) project" data-md-component="logo">
      
  <img src="../images/apperta-logo.png" alt="logo">

    </a>
    Apperta CKM - NHS Pharmacogenetics (PGx) project
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        Overview
      </a>
    </li>
  

    
      
      
      

  
  
    
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        Pharmacogenetics Data Modelling - A Step Towards Personalised Prescribing
      </a>
      
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../CKM-001-%20Reviewing-an-Archetype-on-CKM/" class="md-nav__link">
        Reviewing an archetype on CKM
      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


<h1 id="pharmacogenetics-data-modelling-a-step-towards-personalised-prescribing">Pharmacogenetics Data Modelling - A Step Towards Personalised Prescribing</h1>
<h2 id="introduction">Introduction</h2>
<p>Pharmacogenomics (PGx) is the study of how genetic variation impacts the way people respond to medicines. PGx can help healthcare professionals make more informed and safer prescribing decisions, by providing live guidance on the optimal medicines and dose for each patient.</p>
<p>Unlike many other types of genomic data, PGx results will be used across the health service by professionals who may not have any experience handling genetic information. PGx data can be generated from relatively simple lab tests that look for common genetic changes in genes which are involved in the metabolism of medicines.</p>
<p>The presence of genetic changes in genes related to medicines metabolism can be used to classify people into different categories known as "metaboliser" status. For example, some people may have genetic changes which reduce the activity of an enzyme, making them a poor metaboliser, meaning they may break down medicines more slowly and may experience increased side effects or toxicity. Others may be ultra-rapid metabolisers, which means they might break down medicines very quickly, necessitating  higher doses or different medicines to achieve the desired effect.</p>
<p>These PGx genes and associated changes in metabolism are relevant to many commonly prescribed medicines, such as anti-depressants, statins, and pain killers. As the prescribing of medicines occurs in several different care settings, access to PGx results is required across healthcare systems. This means that digital solutions to support the implementation of PGx must be interoperable by design and ensure data is not locked into silos.</p>
<h2 id="our-project">Our project</h2>
<p>Our team are developing an open data model for the storage and communication of PGx data. We will use openEHR and FHIR technologies to achieve this, with a view to manage PGx data in a safe, standardised and interoperable way. We are supported by the GA4GH Pharmacogenomics working group. </p>
<p><img alt="" src="https://hackmd.io/_uploads/ByWkGnco3.png" /></p>
<p>Our data model is informed by some existing resources:</p>
<ul>
<li><a href="https://vrs.ga4gh.org/en/stable/">G4AGH Variant model</a></li>
<li><a href="https://cpicpgx.org/genes-drugs/">CPIC Gene-drug interactino tables</a></li>
<li><a href="http://build.fhir.org/ig/HL7/genomics-reporting/background.htmlhttp://build.fhir.org/ig/HL7/genomics-reporting/background.html">FHIR Geonomics Reporting Implementation Guide</a></li>
<li><a href="https://ckm.openehr.org/ckm/projects/1013.30.50">openEHR Genomics Project</a></li>
</ul>
<p>However, we have also identified some specific requirements for PGx that are not fully addressed by these resources.</p>
<p>The aim is to provide prescribing advice (clinical decision support) in real-time to GP and other systems, based on established PGx genes and emerging PGx reasearch.</p>
<p><img alt="" src="https://hackmd.io/_uploads/H1r7Mnci3.png" /></p>
<h3 id="key-pgx-concepts">Key PGx concepts</h3>
<h4 id="gene">Gene</h4>
<p>A DNA location responsible for protein manufacture, in this case a set of enzymes responsible for metabolising substances including therapetic agents</p>
<div class="language-text highlight"><pre><span></span><code> CYP2C19
</code></pre></div>
<h4 id="genotype">Genotype</h4>
<p>A more detailed description of the nature of the gene for a specific fragment for a specific patient, in particular some indication of 'variance from normal', which is known to have an impact on the effectiveness of the metaboliser enzyme determined by the gene.</p>
<div class="language-text highlight"><pre><span></span><code>     CYP2C19 
       c.636G&gt;A p.(W212x) 
       c.1279C&gt;T p.(R433W)
</code></pre></div>
<h4 id="haplotype">Haplotype</h4>
<p>A haplotype is a specific way of describing variants, commonly used in PGx, using 'star alleles'.</p>
<div class="language-text highlight"><pre><span></span><code>     *3,*5
</code></pre></div>
<h3 id="diplotype">£ Diplotype</h3>
<p>A diplotype is a genotype term which is a combination of a gene name and haplotype, as an alternative to the nomenlature above</p>
<div class="language-text highlight"><pre><span></span><code>     CYP2C19 *3,*5
</code></pre></div>
<h4 id="pgx-phenotype-metaboliser-status">PGx Phenotype (Metaboliser status)</h4>
<p>In this context we are using 'phenotype' to mean the clinical/real-world impact of the genotype on some aspect of the patient's physiology, usually some sort of change in the associated enzyme's ability to metabolise a group of drugs.</p>
<p>This is gnerally expressed as some sort of change to metaboliser status/function.</p>
<p>CPIC maintains tables that associate a set of PGx phenotypes/mataboliser status codes (LOINC and SNOMED CT) based on known and emerging variants).</p>
<div class="language-text highlight"><pre><span></span><code> CYP2C19 Poor Metaboliser
</code></pre></div>
<h4 id="therapeutic-implications">Therapeutic implications</h4>
<p>On the basis of known PGx phenotypes, specifc therapeutic implications can be derived for specifc drugs or drug classes known to be impacted by a specific PGx phenotype/metaboliser status. 
In some cases that will be to avoid a particular drug/adjust dosage because of reduced metabolic activity and risk of toxicy. 
In other cases, a higher than normal metabolic activity may redice the effectiveness of a particular drug.
The drugs and drug classes that are implicated by a particular PGx phenotype will change over time as new significant variants are detected, or new drugs and drug calasses are developed.</p>
<div class="language-text highlight"><pre><span></span><code>Significantly reduced clopidogrel active metabolite formation;
increased on-treatment platelet reactivity; increased risk for 
adverse cardiac and cerebrovascular events
</code></pre></div>
<h4 id="therapeutic-recommendations">Therapeutic recommendations</h4>
<p>Following from the therapeutic implications, international consensus exists around therapeutic recommendations, which may suggest an alternate medicine or specific dose change. The combination of therapeutic implication and recommendation is also described as 'guidance'. </p>
<div class="language-text highlight"><pre><span></span><code>Avoid standard dose (75 mg/day) clopidogrel if possible. 
Use prasugrel or ticagrelor at standard dose if no contraindication.
</code></pre></div>
<p>This leads to a chain of infomation supporting end-user Clinical Decision support...</p>
<p><img alt="" src="https://hackmd.io/_uploads/r1m6fnqjh.png" /></p>
<h2 id="what-is-reported-in-a-pgx-lab-test">What is reported in a PGx lab test?</h2>
<p>Lab reporting for PGx tests is very variable. In particular, some labs report the test result and therapeutic implications for a specific drug or class of drugs, while others report the metaboliser status for specific tested 
genes, from which therapeutic implications can be derived, normally using guidance from the Clinical Pharmacogenetic Implementation Consortium (CPIC) or the Dutch Pharmacogenetics Working Group (DPWG).</p>
<p>This is one example of a PGX Laboratory report
<div class="language-text highlight"><pre><span></span><code><span id="__span-0-1"><a id="__codelineno-0-1" name="__codelineno-0-1" href="#__codelineno-0-1"></a>{
</span><span id="__span-0-2"><a id="__codelineno-0-2" name="__codelineno-0-2" href="#__codelineno-0-2"></a>  &quot;OrganizationId&quot;: 360,
</span><span id="__span-0-3"><a id="__codelineno-0-3" name="__codelineno-0-3" href="#__codelineno-0-3"></a>  &quot;Identifier&quot;: &quot;204753010015_R06C01&quot;,
</span><span id="__span-0-4"><a id="__codelineno-0-4" name="__codelineno-0-4" href="#__codelineno-0-4"></a>&quot;KnowledgeBase&quot;: &quot;illumina-1-1-01.kb&quot;,
</span><span id="__span-0-5"><a id="__codelineno-0-5" name="__codelineno-0-5" href="#__codelineno-0-5"></a>  &quot;Diplotypes&quot;: [
</span><span id="__span-0-6"><a id="__codelineno-0-6" name="__codelineno-0-6" href="#__codelineno-0-6"></a>    {
</span><span id="__span-0-7"><a id="__codelineno-0-7" name="__codelineno-0-7" href="#__codelineno-0-7"></a>      &quot;Gene&quot;: &quot;CYP2D6&quot;,
</span><span id="__span-0-8"><a id="__codelineno-0-8" name="__codelineno-0-8" href="#__codelineno-0-8"></a>      &quot;ResultType&quot;: &quot;Diplotype&quot;,
</span><span id="__span-0-9"><a id="__codelineno-0-9" name="__codelineno-0-9" href="#__codelineno-0-9"></a>      &quot;Genotype&quot;: &quot;*1/*2&quot;,
</span><span id="__span-0-10"><a id="__codelineno-0-10" name="__codelineno-0-10" href="#__codelineno-0-10"></a>      &quot;Phenotype&quot;: &quot;Normal Metabolizer&quot;,
</span><span id="__span-0-11"><a id="__codelineno-0-11" name="__codelineno-0-11" href="#__codelineno-0-11"></a>      &quot;AllelesTested&quot;: &quot;*1-*90.*1x2N.*1xN.*2.*2x2N.*2xN.*3.*4.*4M.*4N.*4N-*4.*4NxN-*4.*5.*6.*7.*8.*9.*10.*10x2N.*11.*12.*13-*2.*14.*15.*17.*19.*20.*22.*23.*24.*25.*29.*31.*33.*36.*36-*10.*36x2N.*36x2N-*10.*37.*38.*42.*43.*43x2N.*44.*45.*45x2N.*46.*47.*48.*49.*50.*51.*52.*53.*54.*55.*56A.*56B.*57-*10.*59.*62.*64.*68-*4.*70.*71.*73.*81.*82.*84.*86.*87.*88.*89.*90.*94.*95.*96.*99.*100.*101.*103.*106.*107.*108.*109.*110.*111.*112.*113.*114.*115.*116.*117.*121.*123.*125.*126.*131.*132.*134.*135.*136.*3x2N.*4N-*4x2N.*4Nx2N.*6x2N.*9x2N.*17x2N.*29x2N.*102.*4x2N.*4xN.*36-*10x2N.*34&quot;
</span><span id="__span-0-12"><a id="__codelineno-0-12" name="__codelineno-0-12" href="#__codelineno-0-12"></a>    },
</span><span id="__span-0-13"><a id="__codelineno-0-13" name="__codelineno-0-13" href="#__codelineno-0-13"></a>    {
</span><span id="__span-0-14"><a id="__codelineno-0-14" name="__codelineno-0-14" href="#__codelineno-0-14"></a>      &quot;Gene&quot;: &quot;DPYD&quot;,
</span><span id="__span-0-15"><a id="__codelineno-0-15" name="__codelineno-0-15" href="#__codelineno-0-15"></a>      &quot;ResultType&quot;: &quot;ActivityScore&quot;,
</span><span id="__span-0-16"><a id="__codelineno-0-16" name="__codelineno-0-16" href="#__codelineno-0-16"></a>      &quot;Genotype&quot;: &quot;Activity Score: 2&quot;,
</span><span id="__span-0-17"><a id="__codelineno-0-17" name="__codelineno-0-17" href="#__codelineno-0-17"></a>      &quot;Phenotype&quot;: &quot;Normal Metabolizer&quot;,
</span><span id="__span-0-18"><a id="__codelineno-0-18" name="__codelineno-0-18" href="#__codelineno-0-18"></a>      &quot;AllelesTested&quot;: &quot;85T&gt;C*.703C&gt;T*.2657G&gt;A*.2983G&gt;T*.1905+1G&gt;A*.185642_185645delTCAT*.1679T&gt;G*.2846A&gt;T*.rs17376848*.1601C&gt;T*.496A&gt;G*.rs6670886*.1627T&gt;C*.rs1801160*.557A&gt;G*.1003C&gt;A*.475994del*.1236G&gt;A*.c.61C&gt;T*.c.1129-5923C&gt;G*.c.1156G&gt;T*.c.62G&gt;A*.234-123G&gt;C*&quot;
</span><span id="__span-0-19"><a id="__codelineno-0-19" name="__codelineno-0-19" href="#__codelineno-0-19"></a>    },
</span><span id="__span-0-20"><a id="__codelineno-0-20" name="__codelineno-0-20" href="#__codelineno-0-20"></a>    {
</span><span id="__span-0-21"><a id="__codelineno-0-21" name="__codelineno-0-21" href="#__codelineno-0-21"></a>      &quot;Gene&quot;: &quot;MTHFR&quot;,
</span><span id="__span-0-22"><a id="__codelineno-0-22" name="__codelineno-0-22" href="#__codelineno-0-22"></a>      &quot;ResultType&quot;: &quot;Genotype&quot;,
</span><span id="__span-0-23"><a id="__codelineno-0-23" name="__codelineno-0-23" href="#__codelineno-0-23"></a>      &quot;Genotype&quot;: &quot;c.1286A&gt;C AC&quot;,
</span><span id="__span-0-24"><a id="__codelineno-0-24" name="__codelineno-0-24" href="#__codelineno-0-24"></a>      &quot;Phenotype&quot;: &quot;&quot;,
</span><span id="__span-0-25"><a id="__codelineno-0-25" name="__codelineno-0-25" href="#__codelineno-0-25"></a>      &quot;AllelesTested&quot;: &quot;c.665C&gt;T.c.1286A&gt;C&quot;
</span><span id="__span-0-26"><a id="__codelineno-0-26" name="__codelineno-0-26" href="#__codelineno-0-26"></a>    },
</span></code></pre></div></p>
<p>However, we are aware that this type of reporting varies hugely, depending on the lab and analyser technology used.</p>
<p>Our intent is to align with work by the <a href="http://build.fhir.org/ig/HL7/genomics-reporting/background.htmlhttp://build.fhir.org/ig/HL7/genomics-reporting/background.html">FHIR Geonomics Reporting Implementation Guide</a>, which has input from NHS England colleagues, as we anticipate that the industry will gradually converge on this or similar work. </p>
<h2 id="what-do-we-need-to-record-in-the-patient-record">What do we need to record in the patient record?</h2>
<p>A key requirement is to record the PGx phenotype 'metaboliser status' for each PGx gene, and not just end-reult therapeutic implications. </p>
<p>This is because the metaboliser status is a more stable and reliable indicator of drug response than any resultant recommendations, which may change over time, based on emerging research.</p>
<p>Typically PGx testing is targetted at specific genes but coverage of the alleles with those genes is often partial, and is wholly dependent on the specific test used different labs may test against a different range of significant alleles for any particular gene and no single test may be 'complete'. </p>
<p>This gives rise to the need to capture or create a 'PGx profile' which is essentially an aggregation of the metaboliser states identified by multiple 'PGx tests' for that gene over time.  </p>
<h2 id="representing-the-original-pgx-test-result">Representing the original 'PGx test result'.</h2>
<p>The openEHR Lab Analyte Result archetype is nested inside a Lab Test archetype, which represents the original lab test that includes one or more PGx genes. It is very well aligned with HL7v2 and FHIR approaches to handling other kinds of lab test.</p>
<p>The PGx-specific results of the original lab test are handled by a '' cluster archetype which sits inside the Lab Result analyte 'Other details' slot</p>
<h2 id="the-need-to-record-an-evolving-pgx-profile">The need to record an evolving 'PGx profile'</h2>
<p>The Profile Evaluation archetype contains and aggregate of the individual PGx Analyte Results for all tested genes. </p>
<p>We decided not to use the openEHR <a href="">Precaution</a> archetype for this purpose, because PGx information can also suggest positive indications for some drugs, not just risks, precautions or contraindications.</p>
<p>However, this will be discussed further with the openEHR modelling community. </p>
<h2 id="how-much-detailed-variant-information-needs-to-be-captured">How much detailed variant information needs to be captured?</h2>
<p>Ultimately the metaboliser status and therapuetic implications are derived from understanding the known genetic variants, and there is a need to be able to refer back to this 'source information'. </p>
<p>However, we did not want to overload the patient record with very verbose and structured variant information that would be largely of use to the laboratory or research community and not to frontline clinicians. Rather, linknig back to the source information would be the most appropriate approach, meaning those who wanted to review this detailed data could do so. </p>
<h4 id="ga4gh-vrs-attachment">GA4GH VRS attachment</h4>
<p>The decision instead was to use a simple descriptor along with a link to a G4AGH VRS json file carried within an existing international <a href="">Variant Result</a> archetype.</p>
<p>The G4AGH VRS json file provides a standardised and compact way of describing variants, which can be easily exchanged and validated.</p>
<p>Our data model consists of two main components: a Pharmacogenetics Lab Analyte Result archetype, which represents the result of a single PGx test for one gene; and a Pharmacogenetics Profile Evaluation archetype, which represents the aggregated result of all previous PGx tests for a patient. The Lab Analyte Result archetype contains the gene name, the allele names, the metaboliser status and the therapeutic recommendations for that gene. </p>
<h2 id="integration-with-other-systems">Integration with other systems</h2>
<p>Our data model is designed to be compatible with <a href="http://build.fhir.org/ig/HL7/genomics-reporting/background.htmlhttp://build.fhir.org/ig/HL7/genomics-reporting/background.html">FHIR Genomics Reporting profiles</a>, which can be used to export PGx information to other systems or platforms. </p>
<p>We may need to make some minor adjustments to these profiles to accommodate our specific requirements, but we believe that they are largely suitable for PGx reporting.</p>
<h2 id="summary">Summary</h2>
<p>We hope that our work on PGx data modelling will contribute to advancing personalised prescribing and improving patient outcomes. We welcome feedback and collaboration from anyone who is interested in this topic.</p>





                
              </article>
            </div>
          
          
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs - navigation.tabs.sticky"], "search": "../assets/javascripts/workers/search.208ed371.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.407015b8.min.js"></script>
      
    
  </body>
</html>